Automattic/jetpack
# Launchpad Checklist API: Add start writing checklist and tasks

Related  https://github.com/Automattic/wp-calypso/pull/76095

## Proposed changes:

* We've migrated most launchpad checklist definitions into a backend endpoint.
* The start-writing checklist, however, is an exception
* We add start-writing checklist task and task definitions into `jetpack-mu-wpcom`
### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. Run `bin/jetpack-downloader test jetpack-mu-wpcom-plugin add/start-writing-checklist-and-tasks-to-launchpad-checklist-api` to build and sync this branch to your sandbox. 
2. Sandbox a test site. 
3. Start a new onboarding site by logging out, navigating to wordpress.com/setup/start-writing, and signing up for a new account
4. Go to `https://developer.wordpress.com/docs/api/console/`, select Rest API and wpcom/v2, and input the following: `/sites/YOURSITESLUG/launchpad/checklist?checklist_slug=start-writing`. Confirm you get back an appropriate checklist that matches the status of checklist items for the `start-writing` flow
